docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -123,6 +123,22 @@ an external service called the {package-registry}.
 
 For this to work, the {kib} server must connect to `https://epr.elastic.co` on port `443`.
 
+[discrete]
+[[fleet-errors-tls]]
+== {kib} cannot connect to {package-registry} in air-gapped environments
+
+In air-gapped environments where custom Certificater Authorities (CA) are used, if the custom
+CA is not available for {kib}, you can see error like this one:
```

Comment:
I'd change the word order a bit to shorten the sentence. BTW, I'm not a big fan of "encounter" instead of "see" but we've been asked to avoid using ableist language.